[back] docs: more management commands in the readme (#1528)
tournesol-app/tournesol: backend/README.md

docker exec tournesol-dev-api python manage.py create_dataset
```

`python manage.py watch_account_number`

This command checks if the number of accounts using a trusted email

> domain exceeds a specific threshold on a given date and send an

Reviewer: ```suggestion
domain exceeds predefined thresholds on a given date, and sends
```

> alert on Discord.

Reviewer: ```suggestion
alerts on Discord.
```

It takes 2 optional arguments:

> - `-s` only display the results in stdout and don't send a Discord alert

Reviewer: ```suggestion
- `-s` display the results only in the standard output (no alert sent)
```
